quickevent/app/plugins/Classes/src/classeswidget.cpp

```diff
@@ -106,6 +106,8 @@ class CourseItemDelegate : public QStyledItemDelegate
 
 class CourseCodesTableModel : public qfm::SqlTableModel
 {
+	Q_DECLARE_TR_FUNCTIONS(CourseCodesTableModel)
+private:
```

Comment:
This class is a descendant of `QObject`. The canonical way to support `tr` is the `Q_OBJECT` macro. `Q_DECLARE_TR_FUNCTIONS` is for other classes.

Reply:
Thanks, it is fixed.